hack/make-rules/update.sh

```diff
@@ -40,6 +40,7 @@ BASH_TARGETS=(
 	update-generated-api-compatibility-data
 	update-generated-docs
 	update-openapi-spec
+	update-vanity-imports
```

Comment:
Important change: adds vanity imports to the `update.sh`

Reply:
drop the call here, everything else in this list is updating things enforced by verify scripts, but including this will force changes to possibly irrelevant files when people are making changes like API updates

Reply:
Done.